api_tests/src/actor_test.ts

```diff
@@ -113,7 +170,12 @@ async function newCndActor(role: Role) {
         logger,
         finalConfig
     );
-    const cndStarting = cndInstance.start();
+
+    let cndStarting;
+
+    if (startCnd) {
+        cndStarting = cndInstance.start();
+    }
```

Comment:
`CndActor` actually has a `start` function that starts the instance. Instead of making this configurable here, we could just remove the functionality from this fn and do it outside. Thoughts?

It seems like we are awaiting an undefined value on line 189 now in case `startCnd` is false. Not sure if that does any harm?

Reply:
> CndActor actually has a start function that starts the instance. Instead of making this configurable here, we could just remove the functionality from this fn and do it outside. Thoughts?

I agree that the current code design is not great, however, with the current design we start the blockchain nodes and cnd in parallel.
If we were to move it outside, we would start the blockchain nodes together and then cnd.
For the sake of performance I'd be keen to keep this property. Not sure I can do that if I move it outside.

> It seems like we are awaiting an undefined value on line 189 now in case startCnd is false. Not sure if that does any harm?

I tried and was wondering what would happen. It seemed to work as linux CI tests pass but not mac os. I'll investigate but it does not seem related.